change/@fluentui-web-components-2020-10-20-09-05-46-fix-web-components-test.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Comment:
@xugao since this technically doesn't affect files, ideally this would be a chore and not rev the packages.